praw/models/util.py

```diff
@@ -35,7 +35,7 @@ def permissions_string(
 
 
 @_deprecate_args(
-    "function", "pause_after", "skip_existing", "attribute_name", "exclude_before"
+    "function", "pause_after", "skip_existing", "attribute_name", "exclude_before", "start_after"
```

Comment:
I don't think you want to include this argument as part of the deprecated set.

Reply:
I'm pretty sure it needs to be included. The arguments in the decorator indicate the deprecated order of the arguments while the decorated code indicates the order (e.g., sorted alphabetically) moving forward.

Reply:
Oh, my bad. Thanks for the clarification.

Reply:
Anything else I need to do? Do I need to do something to fix the ubuntu-latest hook?

Reply:
Done and done